content/save-load-arrays.ipynb

```diff
@@ -0,0 +1,323 @@
+{
```

Comment:
<p>I would add an observation that the (binary) npz file is not supposed to be human-readable, but rather fast to read/write. Someone might be confused by trying to peek into the file and finding out it is gibberish. (there's also a typo in workspace, item 1.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>That's true, I've modified this paragraph:</p><p> </p><blockquote>If you look in your current working directory, you should have a new file with the name <code>x_y-squared.npz</code>. This file is a zipped binary of the two arrays, <code>x</code> and <code>y</code>. Let's clear the workspace and load the values back in. This <code>x_y-squared.npz</code> file contains multiple [NPY format](https://numpy.org/doc/stable/reference/generated/numpy.lib.format.html#module-numpy.lib.format) files. The data is saved in [native binary format](https://en.wikipedia.org/wiki/Binary_file) so you cannot read the numbers in a standard text editor or spreadsheet.&nbsp;</blockquote>